net/net.go

```diff
@@ -35,3 +130,9 @@ func IsIPv6CIDRString(cidr string) bool {
 	ip, _, _ := net.ParseCIDR(cidr)
 	return IsIPv6(ip)
 }
+
+// IsIPv6CIDR returns if a cidr is ipv6
+func IsIPv6CIDR(cidr *net.IPNet) bool {
```

Comment:
Do we actually need this, or can the caller simply call `IsIPv6(cidr.IP)`  ?

Reply:
no, I changed it.